etc/hnsd.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=handshake DNS resolver
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+EnvironmentFile=/etc/hnsd.conf
+ExecStart=/usr/bin/hnsd $POOL_SIZE $RS_HOST $UNBOUND $PEER
```

Comment:
Does `$PEER` need a value in `etc/hnsd.conf`? Also, why use env variables like this instead of just writing the arguments in the `ExecStart` line?

Reply:
iirc you can't have arguments for the `ExecStart` line, thus the need for this hack. I agree if there's a better way to handle this we should probably use that.

Reply:
I think you can, at least it works on Ubuntu. Here's my service file for the full node at https://easyhandshake.com

```
[Unit]
Description=hsd
After=network.target

[Service]
Type=simple
Restart=always
RestartSec=1
User=root
ExecStart=/root/n/bin/node /root/hsd/bin/hsd --no-wallet

[Install]
WantedBy=multi-user.target
```

Reply:
OK I found that you can't add args if the args have a space in them. So... we can point `hnsd` to `/etc/hnsd.conf` so it will be easily configurable.